themes/default/layouts/shortcodes/code-filename.html

```diff
@@ -0,0 +1,3 @@
+{{ $file := .Get "file" }}
+<h6 class="nohighlight" style="border-top:2px solid lightgray; padding-top: .5em;margin-top: 5px;font-family: monospace;">{{ $file }}</h6>
+{{- .Inner | markdownify -}}
```

Comment:
We have Tailwind helpers for these things, and should generally stick to them wherever we can, for overall design consistency. (Also threw in a structural tweak just to encapsulate things a bit more.)

```suggestion
<div class="border-solid border-t border-gray-300 pt-2">
    <h6 class="nohighlight">
        <span class="font-mono">{{ $file }}</span>
    </h6>
    <div>
        {{- .Inner | markdownify -}}
    </div>
</div>
```